borderless/explainer.md

```diff
@@ -0,0 +1,512 @@
+3. Holding down ESC → App permissions / settings
+4. Message on re-focus of an app
+
+#### Reasonings for discarding
+
+In general exiting differently from entering would be fairly inconsistent and
+wouldn’t reutilize the existing capabilities much more than the chosen solution.
+The chosen approach was the one preferred by the security & privacy team members
+regarding those security aspects combined with the amount of engineering work
+required.
+
+### New JS APIs
+
+Entering and existing borderless mode using JS APIs. On top of the manifest’s
+display_override “borderless” value and user permission, these APIs would enable
+manual toggling of the borderless mode. Calling the APIs in JS could look
+something like this: `window.requestBorderless()` and `window.exitBorderless()`
+
+#### Reasonings for discarding
+
+This could still be considered as an extension on top of the currently planned
+solution if such features would be considered as useful. Currently there’s no
+use-case for needing these.
+
+## Security Considerations
+
+### Spoofing risks
+
+Giving sites partial control of the title bar leaves room for developers to
+spoof content in what was previously a trusted, UA-controlled region. To
+mitigate the threat of spoofing, the feature is (at least for now) only targeted
+to be available for Isolated Web Apps and users will have to opt-in to the
+feature via a window management permission prompt or via admins’ policies. The
+title bar can be returned via App settings by toggling off the borderless mode
+capability.
+
+### Out-of-scope Navigation
+
+An existing security feature for installed web apps is an indicator of when a
+user has left the declared scope of the app. When a user navigates out of scope,
+a black bar appears between the title bar and the web content, and it includes
+the following information:
+
+- A close button to allow users to easily navigate back into scope
+- A security icon which opens the security info popup when clicked
+- The origin and title of the site
+- In borderless mode, if a user navigates out-of-scope the web app will revert
+  to standalone display mode (with a standalone title bar). When the user
+  navigates back into scope, the standalone title bar will be removed again and
+  the borderless mode enabled.
+
+Example of when out-of-scope, so reverting to the standalone title bar with
+custom tab bar stating the current origin:
+
+![Example of out of scope navigation layout](./images/out-of-scope-navigation.png)
+
+### Opening a popup from a borderless PWA
+
+- Opening a popup to any other origin → The popup should NOT be in borderless
+  mode (security risk if the following pop-up app would be malicious).
+- Opening a popup to itself (same origin) → The popup opens in borderless mode,
+  because the app has already got the permission to run in borderless mode.
+
+### Iframes
+
+- Borderless mode should not be available for iframes embedded inside of an IWA.
```

Comment:
I'm not sure what this could mean (since the display mode is only available for a top-level browsing context, there is no possibility of borderless mode, or any display mode, being available for iframes).

I guess you could just reword it to say "Display modes cannot be enabled on embedded pages", to make it clear this isn't a new requirement we need to add, it's just a fact.